fix for bug in gausswin making it produce wrong results
see https://savannah.gnu.org/bugs/index.php?51657
author: Kenneth Hoste (HPC-UGent)
--- signal-1.3.2/inst/gausswin.m.orig	2017-12-11 14:27:49.593981000 +0100
+++ signal-1.3.2/inst/gausswin.m	2017-12-11 14:30:59.032409000 +0100
@@ -37,7 +37,7 @@
     a = 2.5;
   endif
 
-  w = exp ( -0.5 * ( a/m * [ -(m-1) : 2 : m-1 ]' ) .^ 2 );
+  w = exp ( -0.5 * ( a/(m-1) * [ -(m-1) : 2 : m-1 ]' ) .^ 2 );
 
 endfunction
 
